src/pythermalcomfort/zhang_comfort_model_functions/utilities.py

```diff
@@ -0,0 +1,90 @@
+import numpy as np
+
+# 17 body segment names (The name corresponds to JOS-3 model)
+body_name_list = [
```

Comment:
shall we use dataclasses here? I am very worried of using strings for code maintainability in the future.

Reply:
I agree with you, but is it fine to do that later? 

My first priority for now is to get the latest models working on pythermalcomfort. I will make a new issue about this if you accept this initial pull request.